site/docs/3.5.3/configuration.html

```diff
@@ -3467,1499 +3445,6 @@ <h3 id="security">Security</h3>
 
 <h3 id="spark-sql">Spark SQL</h3>
 
-<h4 id="runtime-sql-configuration">Runtime SQL Configuration</h4>
```

Comment:
There are too many deletions on this one. I had to manually edit the headers for this one.

Reply:
Are you saying that for this file, we need to manually edit it on the old file to modify the header?

Reply:
@gengliangwang I have manually edited this file, could you please double check it?